src/main/java/org/terasology/staticCities/sites/SiteFacet.java

```diff
@@ -39,8 +40,8 @@ public SiteFacet(BlockRegion targetRegion, Border3D border, int uncertainBorder)
         this.uncertainBorder = uncertainBorder;
     }
 
-    public Rect2i getCertainWorldRegion() {
-        return getWorldRegion().expand(new Vector2i(-uncertainBorder, -uncertainBorder));
+    public BlockAreac getCertainWorldRegion() {
```

Comment:
Can we rename this to 
```suggestion
    public BlockAreac getCertainWorldArea() {
```